Xamarin.Forms.Core.UITests.Shared/Utilities/Gestures.cs

```diff
@@ -23,7 +41,7 @@ public static bool ScrollForElement (this IApp app, string query, Drag drag, int
 			// check to see if the element is visible already
 			if (app.Query (elementQuery).Length == 1) {
```

Comment:
We expect that because this `app.Query` returned a result then the second `app.Query` to compute `elementDistanceToDragCenter` should return the same result, but actually it doesn't and the fix is to try `n` more times? If so, did we consider reusing the result of the first `app.Query` to also compute `elementDistanceToDragCenter` instead of calling it a second time?